parsec/backend/postgresql/organization.py

```diff
@@ -61,15 +62,21 @@
 """
 )
 
-
 _q_get_stats = Q(
     f"""
 SELECT
     (
         SELECT COUNT(*)
         FROM user_
```

Comment:
So user stats also count revoked users, isn't it a trouble for the SAAS ?

Reply:
Nope